zwift_offline.py

```diff
@@ -1099,6 +1201,9 @@ def api_profiles_activities_id(player_id, activity_id):
     # For using with upload_activity.py (to upload zoffline activity to Zwift server)
     with open('%s/%s/last_activity.bin' % (STORAGE_DIR, player_id), 'wb') as f:
         f.write(activity.SerializeToString())
+    if server_ip != "localhost" and server_ip != "127.0.0.1":
```

Comment:
I would check if server-ip.txt exists instead. Why would someone create the file if server is on localhost?

Reply:
change it .. I do check that file exist within zwift_upload funciton and send logger.info message if not.